homeassistant/components/roomba/config_flow.py

```diff
@@ -0,0 +1,133 @@
+"""Config flow to configure roomba component."""
+import logging
+
+from roomba import Roomba
+import voluptuous as vol
+
+from homeassistant import config_entries, core
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import callback
+
+from . import CannotConnect, InvalidAuth, async_connect_or_timeout
```

Comment:
```suggestion
from . import CannotConnect, async_connect_or_timeout
```

Since you can't tell the difference please remove InvalidAuth